app/controllers/concerns/ab_experiment.rb

```diff
@@ -0,0 +1,31 @@
+# This module wraps the FieldTest to provide a bit of insulation
+# between the FieldTest implementation and the application logic.
+#
+# @note I really wanted this to be a stand-alone model, but in cases
+#       where we don't have a user, the `field_test` method requires
+#       extensive controller context (by way of FieldTest's
+#       field_test_participants controller method).
+module AbExperiment
+  extend ActiveSupport::Concern
```

Comment:
Purely a personal preference (though a relatively strong one): I'm really not big on controller concerns. You have to add one line for the include, then call `feed_strategy_for(user: ...)` in the code. For the reader, it's easy to miss the include and wonder where this method comes from. This could be avoided by making this a class method on a PORO and calling `AbExperiment.feed_strategy_for` directly wherever we need it. Do we derive any benefits from this being a concern?

Reply:
I'm also not a fan of controller concerns, but I need a controller context.  So I checked to see if there were existing concerns before creating a new one.

I'm very happy to refactor and make a PORO/delegator object.  Which will keep the ApplicationController method name list smaller.